### DISCUSSION TITLE
Postgres Extensions Version Management

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9226] [Feature Requests]
Hey,

currently, there is no transparency into what version of a pg extension is installed, and we can only up- or downgrade when contacting the support. With Supabase now starting to ship breaking changes, users should have full transparency of the current version and be able to up- and downgrade in a self-service to handle these changes properly.

**Comment by thorwebdev** [comment]
This is a great idea, thanks! I've flagged with the team and will have them follow up.
